tools/topology/topology1/sof-acp-rmb-tdm8ch-dmic2ch-nau8825-max98360.m4

```diff
@@ -64,8 +68,8 @@ define(matrix2, `ROUTE_MATRIX(4,
 dnl name, num_streams, route_matrix list
 MUXDEMUX_CONFIG(demux_priv_1, 2, LIST_NONEWLINE(`', `matrix1,', `matrix2'))
 
-# playback DAI is ACPSP using 2 periods
-# Buffers use s16le format, 1000us deadline on core 0 with priority 1
+# playback DAI is ACPHS using 2 periods
+# Buffers use s16le format, 2000us deadline on core 0 with priority 1
```

Comment:
don't mix comment changes with functionality changes, this has to go in a separate patch

Reply:
created seperate patch for comments